easybuild/easyconfigs/p/pandas/pandas-1.5.3-foss-2022a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'pandas'
+version = '1.5.3'
+
+homepage = "https://pypi.python.org/pypi/pandas/"
+description = """pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures
+ and data analysis tools for the Python programming language."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['74a3fd7e5a7ec052f183273dc7b0acd3a863edf7520f5d3a1765c04ffdb3b0b1']
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('numexpr', '2.8.1'),
+    ('Bottleneck', '1.3.7'),
```

Comment:
is 1.3.7, from https://github.com/easybuilders/easybuild-easyconfigs/pull/17584, required? 1.3.4 is already included in SciPy-bundle, and there is already an easyconfig for 1.3.6 in this toolchain (we try to avoid using different versions of dependencies in each toolchain generation)